package.json

```diff
@@ -101,8 +101,8 @@
     "parcel-bundler": "^1.10.1",
     "prettier": "^1.11.1",
     "pretty-quick": "^1.4.1",
-    "react": "16.2.0",
-    "react-dom": "^16.3.0",
+    "react": "16.9.0",
+    "react-dom": "^16.9.0",
```

Comment:
The reason for this update is to get unit tests to pass. I basically copied what was done in this PR: https://github.com/FezVrasta/react-popper/pull/308/commits/a86c9c6366c44acd925443aa65f1f8364900739a